website/docs/cli/commands/refresh.mdx

```diff
@@ -45,9 +45,9 @@ plan file, it doesn't allow selecting a planning mode other than "refresh only",
 and `-auto-approve` is always enabled.
 
 Automatically applying the effect of a refresh is risky, because if you have
-misconfigured credentials for one or more providers then the provider may
+misconfigured credentials for one or more providers then Terraform may
 be misled into thinking that all of the managed objects have been deleted,
-and thus remove all of the tracked objects without any confirmation prompt.
+and thus removes all of the tracked objects without any confirmation prompt.
```

Comment:
```suggestion
causing it to remove all of the tracked objects without any confirmation prompt.
```

Reply:
@laurapacilio  - apologies for delayed response. The suggested changes looks good to me. The flow is smoother :)